### PR TITLE
Documentation section and help page

### PR DESCRIPTION
**Pull request by alexandra-aas**
Replaced the "About" page with a "Documentation" section explaining terminology and basic actions.

screencapture-localhost-3000-pages-outcomes-1451315521029

Added an "FAQ" page to hopefully curb some of the questions going directly to Jon and Stephanie.

screencapture-localhost-3000-pages-faq-1451315447714

Removed the "Contact" link on the header that went directly to Jon's email and replaced it with a "Help" page that attempts to push confused uses to the "Documentation" section and "FAQ" page before contacting Jon directly.

screencapture-localhost-3000-pages-help-1451315669309

### CODE REVIEW

**Comment by alexandra-aas** [comment]
@jdaries I'll have a designer from thoughtbot review this on Monday after vacation since it's a significant amount of code. You'll probably want to read through the documentation to give me feedback since something might have been lost in translation. Also, there is only one frequently asked question right now, so maybe you have some more to add. If not we can hide the page, ask Stephanie for some, or leave as is. We'll also have to make updates to the content as we make other changes across the site, but hopefully this is a good start.

File: app/assets/stylesheets/_documentation.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.documentation-side-nav-link {
+  color: $brown;
+  display: block;
+  font-size: $small-font-size;
+  margin-bottom: $base-spacing;
+  transition: $base-transition;
+
+  &:hover {
```
Comment (creuter) [inline app/assets/stylesheets/_documentation.scss]: Thoughts on adding focus styling too, for improved accessibility?

File: app/assets/stylesheets/_faq.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.faq-row {
+  border-top: $base-border;
+  cursor: pointer;
+
+  &:hover {
```
Comment (creuter) [inline app/assets/stylesheets/_faq.scss]: `&:focus`?

File: app/assets/stylesheets/_layout.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.container {
+  @include outer-container;
+}
+
+.narrow-content {
+  @include shift(2);
+  @include span-columns(8);
+}
+
+.no-records {
```
Comment (creuter) [inline app/assets/stylesheets/_layout.scss]: Do you think the `layout` partial is the best place for this?

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -25,12 +25,7 @@
 @import "assessments";
 @import "subjects";
 @import "assessment_types";
+@import "documentation";
+@import "faq";
+@import "layout";
```
Comment (creuter) [inline app/assets/stylesheets/application.scss]: Is there a good reason not to keep these alphabetized?

File: app/assets/stylesheets/refills/_navigation.scss [hunk 5]
```diff
@@ -112,6 +111,15 @@ header.navigation {
       width: auto;
     }
 
+    &.nav-link-mobile-only {
+      font-size: $small-font-size;
+      line-height: $large-line-height;
+
+      @include media ($horizontal-bar-mode) {
```
Comment (creuter) [inline app/assets/stylesheets/refills/_navigation.scss]: Nice descriptive variable name

File: app/views/pages/assessments.html.erb [hunk 6]
```diff
@@ -0,0 +1,71 @@
+<h1>Assessments</h1>
+
+<p>Assessments are ways to measure to what extent students are achieving the Student Learning Outcomes.</p>
+
+<p>The faculty were surveyed and asked to map the outcomes to their respective undergraduate classes. They were then asked which assessments would be used to measure those outcomes.  For your subject, you will see each assessment listed.  Each assessment will be used to measure how many students meet the outcomes selected by the faculty.</p>
```
Comment (creuter) [inline app/views/pages/assessments.html.erb]: There's an extra space before `Each`.

It'd be easier to read this, and comment on specific things, if the lines wrapped at 80 characters

File: app/assets/stylesheets/_documentation.scss [hunk 7]
```diff
@@ -0,0 +1,86 @@
+.documentation-container {
+  min-height: calc(100vh - #{$navigation-height});
```
Comment (ColeTownsend) [inline app/assets/stylesheets/_documentation.scss]: :clap:

File: app/views/layouts/_head.html.erb [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  <%#
+    Configure default and controller-, and view-specific titles in
+    config/locales/en.yml. For more see:
+    https://github.com/calebthompson/title#usage
+  %>
+  <title><%= title %></title>
+  <%= stylesheet_link_tag "application", "data-turbolinks-track" => true %>
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= csrf_meta_tags %>
+  <link href="//fonts.googleapis.com/css?family=Roboto:400,700,300,400italic,700italic,300italic" rel="stylesheet" type="text/css">
```
Comment (ohaicristina) [inline app/views/layouts/_head.html.erb]: I believe you can combine these google fonts into one line using `|`'s
`<link href="//fonts.googleapis.com/css?family=Roboto:400,700,300,400italic,700italic,300italic|Roboto+Condensed:400,700....`

**Comment by ohaicristina** [comment]
:sheep:

File: app/assets/javascripts/faq.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+$(function() {
+  var faqContainer = $('.faq-row');
```
Comment (creuter) [inline app/assets/javascripts/faq.js]: Any thoughts on using data-roles, or `js-` prefixes, to separate concerns for styling an interaction?

**Comment by creuter** [comment]
:shaved_ice:

File: app/assets/stylesheets/refills/_navigation.scss (hunk 5 above)
Comment (creuter) [inline app/assets/stylesheets/refills/_navigation.scss]: Could go without the space after `media`, though

**Comment by ColeTownsend** [comment]
:+1:

**Pull request merged**